.wa-env

```diff
@@ -7,6 +7,8 @@ export OPEN_ID_IDAM_URL=http://sidam-simulator:5000
 export IDAM_USER_URL=http://localhost:5000
 export POSTGRES_PASSWORD=ccd
 export POSTGRES_USERNAME=ccd
+export POSTGRES_ADMIN_PASSWORD=postgres
```

Comment:
While other passwords can be removed, this one can stay here. No objection. Writing just for confirmation that this one is ok.